src/app.js

```diff
@@ -67,6 +69,9 @@ var start = function start(){
         if (process.platform !== 'darwin') {
             app.quit();
         }
+
+        // Testing quit functionality on Mac
```

Comment:
confirm with rak if it works.

Reply:
Seems to be working

Reply:
I would like to leave this unchanged. In the future, If any Mac-specific issue arises, we could come back to this piece of code.